velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -37,6 +37,10 @@ MATCHER(IsInf, "is Infinity") {
   return arg && std::isinf(*arg);
 }
 
+double roundToTwoDecimals(double value) {
```

Comment:
This function can be made local to the test function (inverseWeibullCdf) using it. As its not used anywhere else.